.gitignore

```diff
@@ -3,3 +3,4 @@ Thumbs.db
 .idea/
 cover*
 testdata/tmp/*
+go.sum
```

Comment:
I'd rather not complexify the PR with adding module support. Can you remove this and make it in another PR?

Reply:
Removed additions of go.mod, .gitignore and Makefile - will leave for future PR.